granulate_utils/node.py

```diff
@@ -0,0 +1,6 @@
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+
+DETECTED_NODE_PROCESSES_REGEX = r"^.+/libnode\.so"
```

Comment:
There is no such thing as `libnode`. Node process identification is done by `/proc/pid/exe`.

Reply:
Apparently it's only loaded on some versions. Changed to gProfiler's method, so now all the code is copied.